recruitmentCog/recruitment.py

```diff
@@ -0,0 +1,226 @@
+import asyncio
+import datetime
+import re
+from typing import List
+from datetime import timedelta
+
+import discord
+from redbot.core import Config, checks, commands
+from redbot.core.bot import Red
+from redbot.core.utils.antispam import AntiSpam
+
+
+def sanitize_input(input_text: str) -> str:
+    """Sanitize input to remove mentions, links, and special characters."""
+    sanitized_text = re.sub(r'<@!?&?(\d+)>', '', input_text)
+    sanitized_text = re.sub(r'http\S+', '', sanitized_text)
+    sanitized_text = re.sub(r'([^\w\s.,?!`~@#$%^&*()_+=-])', '', sanitized_text)
+    return sanitized_text
+
+
+class Recruitment(commands.Cog):
+    """A cog that lets a user send a membership application."""
+
+    def __init__(self, bot: Red):
+        self.bot = bot
+        self.message: str = ''
+        self.config = Config.get_conf(self, identifier=101101101101001110101)  # Replace with your own unique identifier
```

Comment:
would love to see this as an env variable

Reply:
This value actually isn't sensitive to Kanium, its just a random identifier that it needs. This will always work no matter where and how its used.